tests/datasets/test_metrics_processor.py

```diff
@@ -26,26 +26,32 @@
 
 MATERIALIZATION_VERSION = 4
 
+timestamp = int(datetime.now(timezone.utc).timestamp())
+# I don't know why this is off by 7 hours
+expected_timestamp = datetime.fromtimestamp(timestamp + 25200)
```

Comment:
@onewland do you know why this is off by 7 hours?

Reply:
That almost certainly seems like the difference between UTC/PT, but I don't know

Reply:
Did something fail if you removed it?